src/library/Server/Manager/Directadmin.php

```diff
@@ -424,43 +521,56 @@ private function _request($command, $fields = array(), $post = true)
             $data = $request->getContent();
         } catch (\Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface |
                  \Symfony\Contracts\HttpClient\Exception\HttpExceptionInterface $error) {
-            $e = new Server_Exception('HttpClientException: :error', [':error' => $error->getMessage()]);
-            $this->getLog()->err($e);
-            throw $e;
+            $exception = new Server_Exception('HttpClientException: :error', [':error' => $error->getMessage()]);
+            $this->getLog()->err($exception);
+            throw $exception;
         }
 
-        if(strlen(strstr($data, 'DirectAdmin Login')) > 0) {
+        if(strlen(strstr($data, '<!doctype html>')) > 0 || strlen(strstr($data, 'DirectAdmin Login')) > 0) {
             throw new Server_Exception('Failed to connect to the :type: server. Please verify your credentials and configuration', [':type:' => 'DirectAdmin']);
         }
 
         if(strlen(strstr($data, "The request you've made cannot be executed because it does not exist in your authority level")) > 0) {
             throw new Server_Exception('Server Manager DirectAdmin Error: "The request you have made cannot be executed because it does not exist in your authority level"');
         }
 
-        $r = $this->_parseResponse($data);
+        $response = $this->_parseResponse($data);
 
-        if(isset($r['error']) && $r['error'] == 1) {
-            $placeholders = [':action:' => $command, ':type:' => 'DirectAdmin'];
-            throw new Server_Exception('Failed to :action: on the :type: server, check the error logs for further details', $placeholders);
+        if(isset($response['error']) && $response['error'] == 1) {
+            $this->getLog()->err('Failed to ' . $command . ' on the DirectAdmin server: ' . $response['text'] . ': ' . $response['details']);
+            throw new Server_Exception('Failed to complete the operation on the DirectAdmin server');
```

Comment:
Any reason not to use the original message?
All of these are turned into translation strings so it's preferable to not create a bunch of unique ones unless there's a specific reason

Reply:
Ah, oversight. Thanks for flagging!